incubator/elasticsearch/templates/client-svc.yaml

```diff
@@ -12,6 +12,9 @@ spec:
   ports:
     - port: 9200
       targetPort: http
+  ports:
```

Comment:
Looks like a copy and paste error, shouldn't have ports again.

Reply:
Also I think when you include multiple ports in a svc you need to add a name value to each of them.

Reply:
thanks for the catch, :+1: